src/App.js

```diff
@@ -1,24 +1,12 @@
-import logo from './logo.svg';
+
 import './App.css';
+import ContactsTable from './components/ContactsTable';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+    <ContactsTable> </ContactsTable>
```

Comment:
```suggestion
    <ContactsTable />
```
Si no le vas a pasar children mejor hazlo asi vale ? :D